app/src/main/java/com/example/harvestfresh/MainActivity.java

```diff
@@ -48,4 +52,28 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         });
         bottomNavigationView.setSelectedItemId(R.id.mHome);
     }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu_main, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
+        if(item.getItemId() == R.id.mLogout){
```

Comment:
Legibility nit:  if (item.getItemId() == R.id.mLogout) {
(Space after "if" and before bracket)